### PR TITLE
Fix bundler config

### PR DESCRIPTION
**Pull request by dkubb**

### CODE REVIEW

**Comment by dkubb** [comment]
@mbj this is ready for review.

File: alpine-ruby/Dockerfile [hunk 1]
```diff
@@ -29,9 +29,9 @@ RUN echo 'gem: --no-document' > ~/.gemrc \
 
 # Setup bundler for the root user
 RUN bundle config --global -- build.nokogiri '--use-system-libraries' \
-  && bundle config --global -- frozen 'true' \
+  && bundle config --global -- frozen '1' \
```
Comment (mbj) [inline alpine-ruby/Dockerfile:32]: Interesting that `true` was not accepted and you had to change it to `1` but it did NOT error out.

Comment (dkubb) [inline alpine-ruby/Dockerfile:32]: Technically `bundler` treats them the same, but when you use `bundle --deployment` it sets this to `'1'` under the hood, so I thought it was important to stick to those semantics.

Comment (mbj) [inline alpine-ruby/Dockerfile:32]: Yeah, this is likely the better tested code path. The fact they support both makes me worry so we should stick to the more likely not to break one.

**Pull request merged**